wordvecspace/server.py

```diff
@@ -145,15 +145,14 @@ def get_vectors(
         get_vectors(["hi", "inidia"]) => [[[ 0.6342  0.2268 -0.3904  0.0368  0.6266], [ 0.      0.      0.      0.      0.    ]]
         """
 
-        return self.wv.get_vectors(
-            words_or_indices, normalized=normalized, raise_exc=raise_exc
-        ).tolist()
+        return self.wv.get_vectors(words_or_indices, normalized=normalized).tolist()
 
     def get_distance(
         self,
         word_or_index1: Union[str, int],
         word_or_index2: Union[str, int],
         metric: str = "angular",
+        normalized: bool = True,
```

Comment:
@RamanjaneyuluIdavalapati Shouldn't this be false by default because this is newly being added
CC: @prashanthellina

Reply:
@jaswanth098 Actually it's not the case.

1. We used the flag `normalized` internally earlier to calculate euclidian distance and cosine distance. If `normalized` is True that means we are dealing with unit vectors which is default. Unit vectors are needed for cosine distance and actually vectors needed for Euclidean. If `normalized` is False means we are calculating Euclidean distance which needs actually vectors (unit vectors * mags).

2. We have implemented a feature taking vector as input for get_nearest and we made assumption of that vector as normalized vector at that time. But we may get actually vector too. That's the reason to notify the type of vector we are using `normalized` flag at `get_nearest` method.

Let me know if you don't understand the above points, we can talk in person.